lib/recorder.js

```diff
@@ -161,7 +161,7 @@ module.exports = {
    * @return {Promise<*> | undefined}
    * @inner
    */
-  add(taskName, fn = undefined, force = false, retry = undefined) {
+  add(taskName, fn = undefined, force = false, retry = undefined, timeout = 0) {
```

Comment:
let's make `timeout` undefined as default so we won't accidentlly run it on 0 second